### ISSUE TITLE
xSchedule - Matrix rendering is reversed

### ISSUE THREAD

**Issue by tmeyermn** [xLightsSequencer/xLights#1307]
I have a matrix in xlights defined as 16 strings, 100 nodes per string, 2 strands and starts at the top left.  When I define the same way in xschedule and create a text playlist the text renders backwards and scrolls the opposite direction for example if I say scroll right to left it scrolls left to right.  If I define the matrix in xSchedule to use Top Right then it renders correctly.

**Comment by keithsw1111** [comment]
This seems unlikely. Can you share your files.

**Comment by tmeyermn** [comment]
Let me know which files you need and I can send them.

**Comment by keithsw1111** [comment]
I need your networks, rgbeffects and xschedule files

**Comment by tmeyermn** [comment]
Ok, I attached them.  I have the matrix set to Top-Right in xSchedule but they are defined as Top-Left in xLights.


[xlights.zip](https://github.com/smeighan/xLights/files/2535011/xlights.zip)

**Comment by keithsw1111** [comment]
Which matrix ... and why are you using individual start channels ... I suspect there is something funky in that setup. For the text in xSchedule the channels for the whole matrix must be contiguous. The right matrix is likely ok ... the left matrix who knows what will happen.

**Comment by tmeyermn** [comment]
It is the Office Matrix I was trying to get working.  I did have to make
the channels contiguous in xSchedule to show the text properly.   It works
just fine either way with fseq from xLIghts.  I had them set up as
individual start channels originally 4 years ago and just stuck with it so
it didn't change my channel assignments (lazy).   I can play with it some
more now that the Halloween season is over using contiguous channels and
see if things straighten out :)

**Comment by keithsw1111** [comment]
xSchedule constraint is it must be contiguous channels. It is not a bug it is intentional.

**Comment by tmeyermn** [comment]
Yes, I'm pretty sure that FPP was the same way for overlaying text.  I will change my matrix to use contiguous channels and adjust the controllers as well.  I'll try it again and see if I still have the problem I was seeing.  Thanks Keith :)

**Comment by derwin12** [comment]
Abandoned. If still an issue please create a new ticket with versions and package logs.
